go/multipooler/manager/rpc_consensus.go

```diff
@@ -156,23 +140,40 @@ func (pm *MultiPoolerManager) ConsensusStatus(ctx context.Context, req *consensu
 		}
 	}
 
-	// Get current WAL position
-	walPosition := &consensusdatapb.WALPosition{}
-	if isHealthy {
-		var err error
-		walPosition, err = pm.GetCurrentWALPosition(ctx)
-		if err != nil {
-			pm.logger.Warn("Failed to get WAL position", "error", err)
-		}
+	// Get WAL position and determine role (primary/replica)
+	walPosition := &consensusdatapb.WALPosition{
+		Timestamp: timestamppb.New(time.Now()),
 	}
-
-	// Determine role (primary/replica)
 	role := "replica"
+
 	if isHealthy {
+		// Check if we're in recovery (standby)
 		var inRecovery bool
 		err := pm.db.QueryRowContext(ctx, "SELECT pg_is_in_recovery()").Scan(&inRecovery)
-		if err == nil && !inRecovery {
-			role = "primary"
+		if err == nil {
+			if inRecovery {
+				// On standby: get receive and replay positions
+				var receiveLsn, replayLsn sql.NullString
+				err = pm.db.QueryRowContext(ctx,
+					"SELECT pg_last_wal_receive_lsn(), pg_last_wal_replay_lsn()").
```

Comment:
The reason we want to return both of these is that when we get to candidate selection, we will start using `last_receive_lsn`.